Automattic/wp-calypso
# Command Palette: Track select event when current site is assumed

## Proposed Changes

Fixes a regression that was preventing the `calypso_hosting_command_palette_command_select` event from being tracked on commands when the current site is assumed.

## Testing Instructions

- Use the Calypso live link below
- Go to a site-specific page (e.g. `/home/:site`)
- Open the command palette
- Select a command with multi-site support (e.g. "Open hosting configuration")
- Make sure the `calypso_hosting_command_palette_command_select` event is tracked
- Make sure the command redirects to the appropriate path for the current site without showing the site selector
- Make sure the `calypso_hosting_command_palette_site_select` event is NOT tracked
- Go to a siteless page (e.g. `/read`)
- Open the command palette
- Select a command with multi-site support (e.g. "Open hosting configuration")
- Make sure the `calypso_hosting_command_palette_command_select` event is tracked
- Make sure the site selector shows up
- Select a site
- Make sure the `calypso_hosting_command_palette_site_select` event is tracked
- Make sure the commands redirects to the appropriate path for the selected site
